modin/utils.py

```diff
@@ -434,6 +437,7 @@ def _documentable_obj(obj: object) -> bool:
         """Check if `obj` docstring could be patched."""
         return bool(
             callable(obj)
```

Comment:
Classes are also callable, but we should not modify them. An important clarification - this only applies to classes that can be stored in the attributes of the class for which we are inheriting docstrings.